sig-usability/charter.md

```diff
@@ -0,0 +1,54 @@
+# SIG Usability Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers
+topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes
+project is maximaly usable by users of a variety of different backgrounds and abilities. Examples
+of efforts include internationalization and accessibility of documentation.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- usability of end-user facing experiences (error messages, end-to-end tasks, etc)
+- accessibility of Kubernetes community artifacts, examples include:
+   - internationalization of documentation in binaries and documentation
+   - color choices for people with color blind-ness
+   - ensuring compatibility with screen reader technology
+- user interface design for core components with user interfaces.
+- translations - https://github.com/kubernetes/kubernetes/tree/master/translations
+- internationalization - https://github.com/kubernetes/kubernetes/tree/master/pkg/kubectl/util/i18n
+
+#### Cross-cutting and Externally Facing Processes
+
+- None currently, eventually perhaps a more formal usability review for changes
```

Comment:
Usability is typically a "cross-cutting" concern. A lot of what they can provide is information for other SIGs to build more useful stuff. useful = usability + utility.

From an "externally facing" standpoint, the CNCF end user community is looking to engage with projects.  Is this something SIG Usability should formally engage in?

Reply:
@mattfarina -- I'm going to be working with @oicheryl to formalize CNCF EUC Comms as a subproject of SIG PM.

Reply:
Suggestion:
#### Cross-cutting and Externally Facing Processes

SIG Usability will facilitate and collaborate with other SIGs on 
- giving end users channels for proving feedback on their experience
- documenting common end user profiles (aka personas)
- documenting common end user interactions with Kubernetes (aka user journeys)
- identifying opportunities for user research and best methods to achieve (qualitative and quantitative).

Reply:
Done.